src/main/java/app/bpartners/gen/annotator/endpoint/rest/model/Annotation.java

```diff
@@ -50,6 +50,9 @@ public class Annotation implements Serializable {
   public static final String JSON_PROPERTY_POLYGON = "polygon";
   private Polygon polygon;
 
+  public static final String JSON_PROPERTY_COMMENT = "comment";
```

Comment:
Did you add this manually or did you generate the java client from the openAPI generator and paste the code here ?

Reply:
It is from OpenAPI generator and I pasted there